component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1394,6 +1394,7 @@ private String importUser(UserImportResultEntity userImportResultEntity,
       return userName;
     }
     boolean onboardUser = !userObject.isNull("onboardUser") && userObject.getString("onboardUser").equals("true");
+    boolean enabledUser = !userObject.isNull("enabled") && userObject.getString("enabled").toLowerCase().equals("false");
```

Comment:
We may rename the variable to be more precise with the value that it will get
```suggestion
    boolean userStatus = !userObject.isNull("enabled") && ("true".equalsIgnoreCase(userObject.getString("enabled") || "false".equalsIgnoreCase(userObject.getString("enabled"));
```